modules/admin_manual/pages/installation/letsencrypt/nginx.adoc

```diff
@@ -1,27 +1,50 @@
 = Setup NGINX
 :toc: right
 :toclevels: 1
+:diffie-hellman-url: https://en.wikipedia.org/wiki/Diffie–Hellman_key_exchange
+:forward-secrecy-url: https://scotthelme.co.uk/perfect-forward-secrecy/
+:ssl_dhparam: http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_dhparam
+:what_is_dh: https://en.wikipedia.org/wiki/Diffie–Hellman_key_exchange
+:letsencrypt-url: https://letsencrypt.org/getting-started/
 
 == Introduction
 
 The following is an example setup process for NGINX, please adapt it to your exact needs.
 
-[[nginx-ssl_dhparam]]
-== NGINX ssl_dhparam
+== Dependencies
+
+To follow this guide, your server needs to have the following dependencies installed:
+
+- Nginx 1.4.4 or later 
+- OpenSSL 1.0.2 or later
+- {letsencrypt-url}[Let's Encrypt]
+
+== Assumptions
+
+This guide assumes two things:
+
+. That you are using Ubuntu Linux 18.04. 
+  If you are not using Ubuntu 18.04, please adjust the instructions to suit your distribution or operating system.
+. That the nginx server configuration file is stored under `/etc/nginx/sites-available/` and is enabled.
+  Not all distributions use this location, however. Please refer to your distribution's Nginx documentation, to know where to store yours. 
```

Comment:
Please separate the list from the following content.

Reply:
That sentence is part of the item as it is above and we have the same in apache.